easy_my_coop_online_payment/__manifest__.py

```diff
@@ -20,13 +20,13 @@
 ##############################################################################
 {
     "name": "Easy My Coop Online Payment",
-    "version": "1.0",
+    "version": "9.0.1.0.0",
```

Comment:
@remytms should this version start with `12`?

Reply:
this one shouldn't be migrated as it is. we should remove it from the 12 branch as it should be completely refactored and get ride of paypal.

@remytms could you just throw it away from 12.0 branch ?

Reply:
Done.